api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -3,8 +3,6 @@
 
 /*
 
-Copyright 2020 The Fluid Authors.
```

Comment:
No need to change in this file.

Reply:
This is auto-generated every time make build. I think we should not add Copyright info here.